src/NexusMods.App.UI/Pages/Sorting/LoadOrder/LoadOrderViewModel.cs

```diff
@@ -7,12 +9,23 @@
 using NexusMods.Abstractions.UI;
 using NexusMods.App.UI.Controls;
 using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
 
 namespace NexusMods.App.UI.Pages.Sorting;
 
 public class LoadOrderViewModel : AViewModel<ILoadOrderViewModel>, ILoadOrderViewModel
 {
     public string SortOrderName { get; }
+    
+    // TODO: Populate these properly
+    public string InfoAlertTitle { get; } = "";
+    public string InfoAlertHeading { get; } = "";
+    public string InfoAlertMessage { get; } = "";
+    [Reactive] public bool InfoAlertIsVisible { get; set; } = false;
+    public ReactiveCommand<Unit, Unit> InfoAlertCommand { get; } = ReactiveCommand.Create(() => { });
+    public string TrophyToolTip { get; } = "";
+    [Reactive] public ListSortDirection SortDirectionCurrent { get; set; } = ListSortDirection.Ascending;
+    [Reactive] public bool isWinnerTop { get; set; } = true;
```

Comment:
Should this be capitalized as `IsWinnerTop` ?

Reply:
Yeah, missed that good catch